daemon/media_player.c

```diff
@@ -1475,6 +1527,97 @@ static mp_cached_code __media_player_add_file(struct media_player *mp,
 	return MPC_OK;
 }
 
+static bool __media_player_open_audio_raw_rtp_file(struct media_player *mp, media_player_opts_t opts) {
+    // Validate codec
+    if (!opts.audio_raw_rtp_codec.len) {
+        ilog(LOG_ERR, "Raw RTP playback requires codec specification");
+        return false;
+    }
+
+    // Convert file path
+    char file_path[PATH_MAX];
+    snprintf(file_path, sizeof(file_path), STR_FORMAT, STR_FMT(&opts.audio_raw_rtp_file));
+    
+    // Open file
+    FILE *f = fopen(file_path, "rb");
+    if (!f) {
+        ilog(LOG_ERR, "Failed to open raw RTP file: %s", file_path);
+        return false;
+    }
+    
+    // Get file size
+    fseek(f, 0, SEEK_END);
+    long file_size = ftell(f);
+    fseek(f, 0, SEEK_SET);
+    // Read entire file into memory
+    char *file_data = malloc(file_size);
```

Comment:
We tend to use `g_malloc` and `g_free` in the code.

Also glib provides a convenient `g_file_get_contents()` helper to do just this.